config/lobby/db/001_create_tables

```diff
@@ -1,94 +1,50 @@
 create table bad_words (
-    word character varying(40) not null
+    word character varying(40) not null primary key
```

Comment:
Shouldn't all changes to this file be reverted?

Reply:
We agreed to not change the schema, which is still the same.
I just moved the altering statements into the create statements, so it's just some readability clean-up